airbyte-webapp/src/pages/connections/ConnectionTransformationPage/DbtCloudTransformationsCard.tsx

```diff
@@ -25,6 +25,50 @@ interface DbtJobListValues {
   jobs: DbtCloudJob[];
 }
 
+const DbtCloudErrorBoundary = class extends React.Component {
```

Comment:
This file is getting really full of components - could we split this into a new file to ease readability & file navigation?

Reply:
@ambirdsall not sure if you intend to address this comment still?

Reply:
@josephkmh this is what happens when I make a mental note but not a physical one. Great point, totally slipped my mind 🤦 👍